### DISCUSSION TITLE
To document: StyleX requires Typescript 5+

### DISCUSSION THREAD

**Discussion by daulric** [facebook/stylex#304] [Q&A] [answered]
![Screenshot (91)](https://github.com/facebook/stylex/assets/57965846/b49b213a-ae4f-4c78-9a15-eaa87706d1f9)
I dont understand why i am getting this error

im using react for this project

**Comment by shawngustaw** [comment]
Try changing your import to `import * as stylex from '@stylexjs/stylex';`

**Comment by nmn** [comment]
Just to confirm you're seeing a type error and not a compile error?

**Comment by daulric** [comment]
> Just to confirm you're seeing a type error and not a compile error?

how do i fix this

its actually my first time learning typescript

**Reply by nmn** [reply]
Please try to provide as much detail about your project as possible. No one can help you with the amount of information you have provided.

**Comment by nmn** [comment, marked as answer]
This error is caused when using Typescript 4 or lower. StyleX requires Typescript 5+
